server/sanitize/index.js

```diff
@@ -1,19 +1,21 @@
+/** @format */
+
 /**
  * Encode single characters with backslashes.
+ *
  * @param {string} charStr single-character string.
  * @return {string} backslash escaped character.
- *
  * @copyright (c) 2013, GoInstant Inc., a salesforce.com company.
  * @license See CREDITS.md.
- * @see https://github.com/goinstant/secure-filters/blob/master/lib/secure-filters.js
- *
+ * @see https:
```

Comment:
Url was cut here, bery strange bug.

Reply:
nice catch, that bug may actually exist upstream in jest-docblock.  I'll investigate

Reply:
I've confirmed its a bug that it is unintended behavior of  `jest-docblock`.  I've opened a pr there: https://github.com/facebook/jest/pull/4837